docs/content/stable/manage/upgrade-deployment.md

```diff
@@ -1,137 +1,169 @@
 ---
-title: Upgrade a deployment
-headerTitle: Upgrade a deployment
-linkTitle: Upgrade a deployment
-description: Upgrade a deployment
+title: Upgrade YugabyteDB
+headerTitle: Upgrade YugabyteDB
+linkTitle: Upgrade YugabyteDB
+description: Upgrade YugabyteDB
 headcontent: Upgrade YugabyteDB on your deployment
 menu:
-  stable:
+  preview:
```

Comment:
Presumably everything else is the same, and edits in preview can be copied to stable - however this line needs to stay set to 'stable'